tools/src/post-mortem/Options.hs

```diff
@@ -54,12 +55,16 @@ throughputOptions = Throughput <$> (argument auto
                                     <> help "directories containing the log files"
                                     )))
 
+testPostProcess :: Parser Options
+testPostProcess = TestPostProcess <$> (argument str (metavar "LOGDIR" <> help "directory containing the log files"))
```

Comment:
I might be wrong (playing this in my head), but cannot you use `strOption` ( or `strArgument`) here directly?

Reply:
was using all of the other options in this file as an example, havent seen this library before